lib/cfa/augeas_parser.rb

```diff
@@ -218,13 +278,15 @@ def serialize(data)
       end
     end
 
-    # Returns empty tree that can be filled for future serialization
+    # @return [AugeasTree] an empty tree that can be filled
+    #   for future serialization
     def empty
       AugeasTree.new
     end
 
   private
 
+    # @param aug [::Augeas]
```

Comment:
I usually do not comment internal methods, but it is fine for me to have there comment, just not important for me.